app/views/talks/_talk.html.erb

```diff
@@ -43,14 +43,36 @@
     <div class="flex flex-col gap-1">
       <div class="flex justify-between">
         <div class="font-bold text-xl">
-          <%= talk.title %>
+          <span><%= talk.title %></span>
         </div>
 
-        <div class="tooltip tooltip-top" data-tip="You can suggest some modifications to this talk">
-          <%= link_to edit_talk_path(talk), class: "flex gap-2 items-center rounded-full border bg-gray-100 hover:bg-gray-200/50 border-gray-200 font-regular px-3 py-1 text-sm" do %>
-            <%= heroicon :pencil, size: :sm %>
-            <span>Edit</span>
+        <div class="flex gap-1">
+          <% if signed_in? %>
+            <% if talk.watched? %>
+              <div class="tooltip tooltip-top" data-tip="Mark talk as unwatched">
+                <%= button_to [talk, :watched_talk],
+                      class: "flex gap-2 items-center rounded-full border bg-gray-100 hover:bg-gray-200/50 border-gray-200 font-regular px-3 py-1 text-sm",
+                      method: :delete do %>
+                  <%= heroicon :x_mark, size: :sm %>
+                  <span class="text-nowrap">Unwatched</span>
```

Comment:
Also, maybe just "Unwatch" here for the text. Because "Unwatched" could be seen that it's currently in the "Unwatched" state which technically means the exact opposite 😅